<ui version="4.0" >
 <class>AFindDialogClass</class>
 <widget class="QDialog" name="AFindDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>184</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Find Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="upFindLabel" >
       <property name="text" >
        <string>Find:</string>
       </property>
       <property name="buddy" >
        <cstring>upFindCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="upFindCombo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="2" row="0" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="upFindButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Find</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upCloseButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="2" row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="upOptionsGroup" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="upWordsCheckBox" >
          <property name="text" >
           <string>Match &amp;Words</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="upCaseCheckBox" >
          <property name="text" >
           <string>Match &amp;Case</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="upRegExpCheckBox" >
          <property name="text" >
           <string>&amp;Regular Exp.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="upRegionGroup" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Region</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QRadioButton" name="upSelectRadio" >
          <property name="text" >
           <string>&amp;Selection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="upToEndRadio" >
          <property name="text" >
           <string>&amp;To End</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="upAllRadio" >
          <property name="text" >
           <string>&amp;All Text</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" >
    <widget class="QGroupBox" name="upDirectionGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Direction</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="upUpRadio" >
        <property name="text" >
         <string>&amp;Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="upDownRadio" >
        <property name="text" >
         <string>Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="upStatusLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
